docs/pages/project/contributing/contributing-ui-tests.md

```diff
@@ -133,7 +133,65 @@ In the last step go to ui folder,
 cd ui;
 ```
 
-And run this command for running the test cases:
+## Run the test cases
+
+There are several options we can use to run the test cases, in CLI:
+
+To run playwright UI mode using the browser, you can add `--ui` in the cli, for example:
+```bash
+npx playwright test --ui
```

Comment:
curious if there are any other steps required. When i just run this command. I get a failling env test unless i fix up the variables. Do we have docs to clarify for new contributors here?

Reply:
We have it under `Setting up environment variable` section above